.travis.yml

```diff
@@ -9,16 +9,19 @@ jobs:
       stage: test
       compiler: clang
       python: 2.7
+      node_js: 7
       addons:
         apt:
           sources: ['ubuntu-toolchain-r-test']
-          packages: ['cmake', 'nodejs', 'g++-5']
+          packages: ['cmake', 'g++-5']
       before_install:
         - export CC="${CC_COMPILER}"
         - export CXX="${CXX_COMPILER}"
         - export ASAN_OPTIONS="symbolize=1"
       install:
         - pip install --user flake8
+        - "curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -"
```

Comment:
Interesting bug. I verified it's in node 0.10.0 but not 0.10.48, so it was fixed during that release.

Ok, I guess it makes sense to use newer node. Did you verify downloading is the best way to do so? Looks like they have [options to use newer node](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/) (but that might be for node-only projects? not sure).

Also please add a comment saying why we need a newer node, with a TODO to remove it (hopefully when travis supports 16.04).